src/pages/DatasetRegistration.js

```diff
@@ -637,18 +700,25 @@ class DatasetRegistration extends Component {
                         ]),
                       ]),
                     div(
-                      {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group rp-last-group'},
+                      {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'},
                       [
-                        input({
-                          type: 'text',
+                        h(Select, {
+                          key: fp.isEmpty(this.state.datasetData.dac) ? null : this.state.datasetData.dac.value,
                           name: 'dac',
                           id: 'inputDac',
-                          maxLength: '256',
-                          value: this.state.datasetData.dac,
-                          onChange: this.handleChange,
+                          value: this.state.datasetData.dac.label,
+                          onChange: (option) => this.onDacChange(option),
+                          blurInputOnSelect: true,
+                          openMenuOnFocus: true,
+                          isDisabled: false,
+                          isClearable: true,
+                          isMulti: false,
+                          isSearchable: true,
+                          options: this.dacOptions(),
+                          placeholder: 'Select a DAC...',
                           className: (fp.isEmpty(this.state.datasetData.dac) && showValidationMessages) ?
-                            'form-control required-field-error' :
-                            'form-control',
+                            'required-field-error' :
+                            '',
```

Comment:
The required field error label doesn't clear state after being invalid, then selecting a valid one.

![Screen Shot 2020-10-07 at 2 52 08 PM](https://user-images.githubusercontent.com/116679/95375057-17682e00-08ad-11eb-86f8-889c0aa44658.png)